HabitRPG/TableViewController/Inventory/Shops/HRPGShopViewController.h

```diff
@@ -6,9 +6,9 @@
 //  Copyright © 2016 Phillip Thelen. All rights reserved.
 //
 
-#import "HRPGBaseViewController.h"
+#import "HRPGCollectionViewController.h"
 
-@interface HRPGShopViewController : HRPGBaseViewController <UIAlertViewDelegate>
+@interface HRPGShopViewController : UIViewController <UIAlertViewDelegate>
```

Comment:
I think I originally had this as a `HRPGCollectionViewController` but then decided against it... I'm still working this part out.